strawberryfields/backends/fockbackend/circuit.py

```diff
@@ -796,4 +797,5 @@ def measure_homodyne(self, phi, mode, select=None, **kwargs):
         # Normalize
         self._state = self._state / self.norm()
 
-        return homodyne_sample
+        # `homodyne_sample` will always be a single value since multiple shots is not supported
+        return np.array([[homodyne_sample]])
```

Comment:
👍

Reply:
hmm, out of scope but find it odd that it's a single mode here whereas it was a list of modes for `measure_fock`. do you perhaps know why this difference is there?